coverage/files.py

```diff
@@ -489,6 +489,9 @@ def map(self, path: str, exists:Callable[[str], bool] = source_exists) -> str:
 
         # If we get here, no pattern matched.
 
+        if self.relative:
```

Comment:
A key indication of whether the test is doing what it should:  comment out "the fix" and see if the test fails.  When I comment out these two lines, all the tests still pass.